packages/view-util/query/dimensionIdentifiers.rq

```diff
@@ -0,0 +1,8 @@
+PREFIX schema: <http://schema.org/>
+PREFIX cube: <https://cube.link/>
+
+select ?dimension ?type (lcase(?id) as ?identifier) {
+  ?dimension a ?type ; schema:identifier ?id
+
+  FILTER ( ?type in (cube:KeyDimension, cube:MeasureDimension) )
+}
```

Comment:
We need this to distinguish measures from other dimensions. The latter will have `_uri` appended to their identifier

Reply:
I'm a little lost here, 

Why add the information of being a measure in the identifier itself?
 Wouldn't be having the type MeasureDimension enough?